pallets/external-validators/src/lib.rs

```diff
@@ -380,8 +382,10 @@ pub mod pallet {
         ) -> DispatchResult {
             // If more validators than max, take the first n
             let validators = BoundedVec::truncate_from(validators);
-            <ExternalValidators<T>>::put(validators);
+            <ExternalValidators<T>>::put(validators.clone());
```

Comment:
```suggestion
            <ExternalValidators<T>>::put(&validators);
```

Reply:
Updated in [13a848e](https://github.com/moondance-labs/tanssi/pull/857/commits/13a848e194fcd9666c423f5590a39061b9cbc37a)